src/Ombi/ClientApp/app/search/moviesearch.component.html

```diff
@@ -2,7 +2,7 @@
 <div role="tabpanel" class="tab-pane active" id="MoviesTab">
 
     <div class="input-group search-bar-background">
-        <input id="search" type="text" class="form-control form-control-custom form-control-search form-control-withbuttons"
+        <input id="search" type="text" placeholder="Type Here to Search" class="form-control form-control-custom form-control-search form-control-withbuttons"
```

Comment:
This ideally should have a translated value and not a hardcoded string
```suggestion
        <input id="search" type="text" placeholder="{{ 'Search.SearchBarPlaceholder' | translate}}" class="form-control form-control-custom form-control-search form-control-withbuttons"
```
You will need to create that in the `en.json` file

Reply:
Added